guide/interactions/autocomplete.md

```diff
@@ -0,0 +1,91 @@
+# Autocomplete
+
+Autocomplete allows you to dynamically provide a selection of values to the user, based on their input. In this section we will cover how to add autocomplete support to your commands.
+
+::: tip
+This page is a follow-up to the [interactions (slash commands) pages](/interactions/slash-commands.md). Please carefully read those first so that you can understand the methods used in this section.
+:::
+
+## Enabling autocomplete
+
+To use autocomplete with your commands, you have to set the respective option when deploying commands:
+
+```js {8}
+const commandData = new SlashCommandBuilder()
+	.setName('autocomplete')
+	.setDescription('test command to show how autocomplete should be setup')
```

Comment:
```suggestion
	.setDescription('Test command to show how autocomplete should be set up')
```

Slight grematical mistake. setup is a noun, "set up" is the verb we're probably looking for